Module 1: Linter Set up for HTML & CSS

ProgramKingAbel opened the pull request:
Set up Linter Working Environment  and HomePage

Configured Linters - Stylelint, LightHouse and Webhint.
Developed homepage Markup and Styling.
Edit professional README.md file
Installed NPM and JSON dependencies
Added MIT LICENSE.TXT file
Fixed broken MIT link in README.md

ProgramKingAbel committed:
Setup a linter for HTML and CSS

ProgramKingAbel committed:
Add stylelint and json file

ProgramKingAbel committed:
Add markup for homepage

ProgramKingAbel committed:
Added styling to index file

ProgramKingAbel committed:
Add test.md file

ProgramKingAbel committed:
Solved Identation on Styles.css

ProgramKingAbel committed:
Add hintrc file

ProgramKingAbel committed:
Update lighthouse version

iambenkis commented on README.md:
```diff
@@ -1 +1,185 @@
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+
```
- [x] Your README file looks very professional 🤙 , however I noticed that the link attached to the MIT license is broken , increase the beauty of your README file by fixing this link with a MIT license.
- [x] You could find it[ here](https://choosealicense.com/licenses/mit/)

ProgramKingAbel replied:
Working Environment set up with Linters - Stylelint, LightHouse and Webhint.
Developed homepage Markup and Styling.
Edit professional README.md file 
Installed NPM and JSON dependencies

ProgramKingAbel replied:
Hello @iambenkis Following the comment on your review, I have added a descriptive summary of my PR as well as added an MIT LICENSE.TXT file to the repository and fixed the broken MIT license broken link in my README.MD. Thank you for the guidance.

iambenkis reviewed (changes requested):
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @ProgramKingAbel   
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

![almost-there](https://user-images.githubusercontent.com/57562869/143511358-9cde7daf-4ad0-4db2-8f15-18b54288ff91.gif)

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 

- [x] Good job opening your first Pull Request however it could be better if your PR have a descriptive title telling you what changes are inside. It will make it more professional 👍 .
- [x] Also your Request should have a short summary describing introduced changes in more detail. Kindly add them on the places showed at the image below

![Capture](https://user-images.githubusercontent.com/80458199/226688486-35003c44-4580-4260-9bc8-04fb1afba43a.JPG)


- [x] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me @iambenkis  when you comment, so I can receive the notification or use Slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

ProgramKingAbel committed:
Update README.md

ProgramKingAbel committed:
Add Profile Links to READMe.md

ProgramKingAbel committed:
Update README.md

ProgramKingAbel commented:
Added LICENSE.TXT file 
Fixed broken License link in README.md

ProgramKingAbel commented:
***@***.*** <https://github.com/iambenkis> I appreciate the Review
feedback. Following that, I added the descriptive title and summary to my
PR as well as added an MIT LICENSE.TXT file to my work and fixed the broken
MIT license link that was in my README.md file. I hope my corrections
satisfy the professional standard that you have advised. Thank you.*

On Tue, Mar 21, 2023 at 6:12 PM Benjamin Kisenge ***@***.***>
wrote:

> ***@***.**** requested changes on this pull request.
> STATUS: REQUIRED CHANGES ♻️♻️
>
> Hi @ProgramKingAbel <https://github.com/ProgramKingAbel>
> Good job so far! I know you worked hard to implement this project, I
> really appreciate the effort you put in! But there are some issues that you
> still need to work on to go to the next project but you are almost there!
>
> [image: almost-there]
> <https://user-images.githubusercontent.com/57562869/143511358-9cde7daf-4ad0-4db2-8f15-18b54288ff91.gif>
> To highlight 🍾
>
>    - All linters are passing ✔️
>    - Good commit history ✔️
>    - Follow GitHub flow ✔️
>
> Required changes ♻️ ♻️
>
>    - Good job opening your first Pull Request however it could be better
>    if your PR have a descriptive title telling you what changes are inside. It
>    will make it more professional 👍 .
>    - Also your Request should have a short summary describing introduced
>    changes in more detail. Kindly add them on the places showed at the image
>    below
>
> [image: Capture]
> <https://user-images.githubusercontent.com/80458199/226688486-35003c44-4580-4260-9bc8-04fb1afba43a.JPG>
>
>    - Check the comments under the review.
>
> Optional suggestions
>
> *Every comment with the [OPTIONAL] prefix is not crucial enough to stop
> the approval of this PR. However, I strongly recommend you to take them
> into account as they can make your code better.*
>
> Cheers and Happy coding!👏👏👏
>
>
> *Feel free to leave any questions or comments in the PR thread if
> something is not 100% clear. Please ping me @iambenkis
> <https://github.com/iambenkis> when you comment, so I can receive the
> notification or use Slack with the same name.*
>
> *Please, do not open a new Pull Request for re-reviews. You should use the
> same Pull Request submitted for the first review, either valid or invalid
> unless it is requested otherwise.*
> ------------------------------
>
> *As described in the Code reviews limits policy
> <https://microverse.zendesk.com/hc/en-us/articles/1500004088561> you have a
> limited number of reviews per project (check the exact number in your
> Dashboard). If you think that the code review was not fair, you can request
> a second opinion using this form <https://airtable.com/shrQAqnBwek5a0O0s>.*
> ------------------------------
>
> In README.md
> <https://github.com/ProgramKingAbel/Hello-Microverse/pull/1#discussion_r1143741261>
> :
>
> > +## 📝 License <a name="license"></a>
> +
> +This project is [MIT](./LICENSE) licensed.
> +
> +
>
>
>    - Your README file looks very professional 🤙 , however I noticed that
>    the link attached to the MIT license is broken , increase the beauty of
>    your README file by fixing this link with a MIT license.
>    - You could find it here <https://choosealicense.com/licenses/mit/>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/ProgramKingAbel/Hello-Microverse/pull/1#pullrequestreview-1351014958>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AH7MZZUGLB52FOSQ2JZDZ4TW5HOQLANCNFSM6AAAAAAWCFXC7U>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

ProgramKingAbel committed:
Add License.txt and fix broken link in README.md

osamaashraf6 reviewed (approved):
Hi @ProgramKingAbel 😊, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
![Congratulations!GIF](https://user-images.githubusercontent.com/83360636/226840539-73d2ae12-9263-46a3-be01-25dcaec40b4a.gif)

## Highlights
- No linter errors present
- The PR has a proper title and a descriptive body 👍

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
